### DISCUSSION TITLE
Need a way to disable CU after auto-updating to 120 :(

### DISCUSSION THREAD

**Discussion by Mazedotexe** [3kh0/ext-remover#1133] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I enabled CAUB on my Chromebook about a year ago, and also remember turning on CAUB flags later just to be safe. Just today though, my Chromebook was on a wifi network it had never been on (this has been true for a few days). I didn't realize anything was wrong, but then when I next turned it on I saw a screen that said not to turn off the Chromebook because it was undergoing a "critical update." This update was going from v114 to v120. So immediately I lost my LTMEAT privileges, and deleted my kill/hang bookmarks because they were now useless. On another computer I took a look at the repo to relearn the other methods, and LTMEAT Flood worked, although it took quite a long time and only sometimes worked. (Later on I switched to Print, quicker and almost always works, still has not expired for me. This isn't surprising, though, because I got it up to 17k pages.) However, there is still one major problem.
    My Chromebook has 2 blocker extensions: GoGuardian and Cisco Umbrella. I already knew this, but Cisco Umbrella does not have an "Allow access to file:// URLs" switch, or any others for that matter, and these are necessary in all of the LTMEAT variations. The only way I can find to bypass CU is to use the chrome://chrome-signin WebView, which I found on this repo but is no longer here. It's one of those nice exploits that are so simple, you feel like you could've found them. However, this is not an adequate solution for a few reasons. First of all, many things are disabled in WebView exploits, and one of them happens to be cookies/localStorage. If I want to access Discord, for example, I need to re-login every time I open one of these tabs. Second of all, there is a big bar at the bottom that will either resize the page to be shorter or downright cover content. This makes the exploit very annoying to use on several levels, and is a massive downgrade to what I literally had earlier today.
![image](https://github.com/user-attachments/assets/4eb7f6bf-b69b-4f43-8313-1ebe5219f9fe)
    So what I am looking for is a safe exploit to turn off Cisco Umbrella that does not involve anything like permanently turning them off, or booting into recovery/developer mode. Skiovox is not an option for reasons I can't go into here, and you couldn't pay me to go back to using proxies. I suppose I could downgrade if I had total certainty it wouldn't go wrong, but I thought I remembered resetting with esc-power-refresh requiring information only my sysadmin would have to bring it back up. I sincerely hope I'm not stuck with a nothing-chromebook from now on, but it's starting to look like it. I will eventually get a laptop, but for now there are lots of things I could only do on my Chromebook before that I'm now unsure of. So, if anybody has something I could use in this situation, it would be _**_greatly_**_ appreciated.
    Also, this is not my first time asking this. A while back I asked how LTMEAT variations would work on Cisco Umbrella, and I got [this](https://github.com/3kh0/ext-remover/discussions/729#discussioncomment-7329032) answer from AshtonDavies.
![image](https://github.com/user-attachments/assets/8f034bf9-ad9e-44fe-9cc5-17a773ff78f6)
Then, I asked out of pure curiosity. Now, I ask out of necessity. I sincerely hope I get a useful response, and if so, thanks in advance...

**Comment by AshtonDavies** [comment]
I have some notices after skimming a good chunk of your essay here. Unfortunately, Cisco Umbrella does in fact have webRequest permissions, so the switch is not skippable.

It is usually necessary to keep a hanged tab open to deal with Manifest V3, according to Bypassi.

**Reply by Mazedotexe** [reply]
Is it possible to find some other obscure way to kill anyways? Or another, _safe,_ exploit?

**Reply by AshtonDavies** [reply]
> Is it possible to find some other obscure way to kill anyways? Or another, _safe,_ exploit?

I mean, if you really want to screw around with overloading pages until they're unresponsive, that is pretty much the only way to kill, but you can't really hang it right after that.

This has been known the limit to the LTMEAT exploit from day one.

**Comment by Perplexedykwhat** [comment, marked as answer]
kill goguardian with ExtHang3r, then use rigtools to disable Cisco Umbrella, if not blocked.

**Reply by Mazedotexe** [reply]
THANK YOU SO MUCH FOR THIS! I'm not sure what ExtHang3r is, and I might check it out later out of interest (not out of necessity because I have LTMeat print) but I had no idea what Rigtools was until this comment. I've been stuck with Cisco Umbrella ever since I made this post, and this is the answer that finally solved it for me. Thank you, SO MUCH!
